src/workerd/tests/BUILD.bazel

```diff
@@ -151,6 +149,12 @@ kj_test(
         "//src/workerd/server:workerd",
     ],
     tags = ["requires-fuzzilli"],
+    # reprl is only compatible with Linux
+    target_compatible_with = select({
```

Comment:
Makes sense but seems totally unrelated?

Reply:
(Am totally fine with including it in this PR, but curious how it came up w.r.t. clang-tidy)

Reply:
clang-tidy has a check that I ended up not including here since it's too spammy, but it complained about __linux__ always being false – which is expected to happen on macOS but not really a problem. I still tried to replace it and avoid building the test on macOS where it's just a no-op, but looks like the target_compatible_with approach would take more work since kj_test already hardcodes a value for that. Dropped this for now.

Reply:
Gotcha, thanks.